vtex/styleguide
# Revert "Update deps" and fix build

#### What is the purpose of this pull request?
This reverts commit 36d3bbf903bae6928c8da500f3b72e20cd384542.



#### What problem is this solving?
This commited updated styleguide dependencies and cause the build to break.


#### How should this be manually tested?
*Remove the `node_modules` to be sure* 
`yarn && yarn styleguide`

See if it builds styleguide correctly

#### Screenshots or example usage

#### Types of changes
